doc/cephfs/fs-volumes.rst

```diff
@@ -292,6 +302,14 @@ List subvolumes using::
 
 .. note:: subvolumes that are removed but have snapshots retained, are also listed.
 
+Check the presence of any subvolume using::
+
+    $ ceph fs subvolume exist <vol_name> [--group_name <subvol_group_name>]
+
+The returned strings by the exist command can be of these two types:
```

Comment:
nit: s/exist/'exist', exist in quotes ?

Reply:
Actually better yet "The strings returned by the `exist` command"